dlk/python/dlk/templates/src/func/matmul.cpp

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2018 The Blueoil Authors. All Rights Reserved.
```

Comment:
Copyright **2019**

Reply:
fixed the license with https://github.com/blue-oil/blueoil/pull/65/commits/a7d5a5b5c0b99405dda466ecbfd0ca2d5dba56ee